tables/uz-g1.utb

```diff
@@ -31,10 +31,12 @@
 #-author-name: BAUM Engineering
 #-author-email: liblouis@baum.ro
 
+include ru-unicode.dis
+
 space \x00a0 a
 include spaces.uti
 
-include ru-unicode.dis
+begcapsword 46
```

Comment:
What worries me about using `begcapsword` is that in the ru-litbrl.ctb table, which you include from uz-g1.utb, capitalization is handled in an entirely custom way. Mixing that approach with opcodes like `begcapsword` seems tricky. Maybe it works fine, because at the bottom there are some rules to cancel out most of the capitalization marks that are added in ru-litbrl.ctb. However it adds another layer of complexity which makes it hard to debug the table when issues arise.

The solution that I would like to see is that the marking of capitals and Cyrillic is extracted from ru-litbrl.ctb into its own file, so that everything in ru-litbrl.ctb _except_ these rules can be included in uz-g1.utb. That would simplify things a lot.

Perhaps @Futyn-Maker has an opinion about this?